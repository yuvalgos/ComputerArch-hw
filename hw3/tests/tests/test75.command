./dflow_calc tests/test75.dat tests/test75.in d188 p191 p190 d110 p68 p158 p117 d125 d20 d164 d32 d41 d140 p84 d90 d108 d154 d106 p59 d158 p57 d122 d193 d3 d43 p190 d141 d87 d10 p103 d132 p156 p60 d121 p91 d36 p7 p63 p69 d36 p43 p153