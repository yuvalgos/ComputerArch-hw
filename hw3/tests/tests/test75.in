6 5 21 17
1 15 25 2
6 3 27 14
6 7 19 10
0 23 1 28
7 20 29 2
6 5 22 9
7 23 1 20
2 3 20 23
4 7 10 15
4 29 19 29
4 15 3 19
2 14 2 29
8 13 26 22
8 14 28 4
4 26 16 12
3 22 22 14
4 24 2 4
6 28 7 10
1 11 16 16
7 7 26 0
4 24 31 21
0 17 1 16
2 21 25 15
8 16 25 28
1 19 10 3
0 31 22 24
0 6 6 25
5 30 14 0
1 23 9 14
7 26 2 21
0 24 11 8
7 21 4 9
3 4 27 26
8 5 3 17
0 30 7 5
6 5 0 26
4 12 19 24
7 31 27 18
7 16 6 24
0 0 0 10
6 16 28 6
4 24 22 16
1 28 13 25
0 27 22 9
2 29 14 5
5 17 17 20
1 22 10 1
4 29 16 5
5 15 25 28
0 26 6 26
1 12 30 19
2 6 19 27
1 29 15 11
8 20 17 22
6 25 21 4
8 16 20 18
8 24 6 4
1 17 14 11
2 26 10 5
7 8 29 0
0 12 13 10
2 14 29 27
0 29 23 2
4 22 14 0
0 30 3 20
7 19 15 18
1 0 10 15
7 6 1 9
2 2 6 11
7 7 20 28
8 5 10 22
8 1 16 13
4 12 7 27
7 12 6 2
1 18 12 24
7 23 23 11
1 30 17 14
3 22 13 6
1 1 8 16
3 22 5 10
3 24 8 20
0 11 20 25
8 19 17 2
7 8 4 31
4 28 28 14
5 12 22 26
6 28 25 4
8 8 4 23
1 24 29 11
6 26 10 22
7 13 11 6
3 1 3 20
4 23 7 15
8 29 14 7
3 23 28 29
2 23 0 19
8 20 15 0
1 11 25 26
8 10 0 14
4 31 22 25
4 0 21 20
6 31 21 26
5 20 20 9
7 11 17 29
4 28 12 12
2 12 26 16
7 26 16 29
5 14 22 6
6 28 22 10
7 1 13 26
5 16 20 5
2 0 18 6
8 0 13 24
4 11 26 17
8 5 31 17
1 15 25 22
2 19 19 8
6 10 19 13
4 24 16 29
2 12 17 23
2 16 9 14
1 9 19 21
0 2 28 28
6 30 8 0
0 11 5 13
5 29 16 26
0 13 24 30
1 20 29 26
7 26 17 7
4 9 29 6
4 0 18 0
7 0 27 15
3 21 2 11
6 20 11 16
5 16 26 31
0 23 0 27
8 26 9 21
7 6 18 4
3 2 1 31
8 16 24 26
4 14 16 2
7 6 30 11
4 5 3 5
3 23 26 4
5 14 29 20
0 17 8 24
0 17 24 26
2 22 5 4
3 4 6 28
8 30 20 28
6 15 15 14
0 26 31 11
7 0 11 14
3 18 2 25
4 3 30 13
3 10 26 19
2 28 16 5
8 10 14 24
4 26 0 0
5 10 29 12
8 8 7 12
0 13 28 24
3 2 16 19
4 20 28 25
4 0 6 31
1 11 4 21
6 0 30 20
4 10 10 26
6 2 21 12
5 24 10 27
7 17 14 27
6 21 22 15
8 13 16 24
0 0 31 31
1 1 5 16
3 12 30 21
0 30 26 22
7 13 1 25
3 9 17 31
8 20 3 29
3 22 4 26
4 16 5 8
2 27 30 12
7 12 21 11
0 3 24 14
6 4 14 15
8 17 28 0
6 30 23 31
7 31 29 11
6 10 8 24
5 14 7 31
8 9 23 9
4 0 11 5
4 1 3 9
3 22 3 26
1 30 20 18
3 30 16 28
0 15 30 4
7 8 26 22
