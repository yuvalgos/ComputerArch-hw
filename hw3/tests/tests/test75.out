Reading the opcodes latency info from tests/test75.dat ... Got latency for 9 opcodes
Reading the program file tests/test75.in ... Found 200 instructions
getProgDepth()==259
getInstDeps(188)=={144,102}
getDepDepth(191)==211
getDepDepth(190)==231
getInstDeps(110)=={91,107}
getDepDepth(68)==0
getDepDepth(158)==177
getDepDepth(117)==108
getInstDeps(125)=={115,91}
getInstDeps(20)=={15,-1}
getInstDeps(164)=={157,55}
getInstDeps(32)=={-1,-1}
getInstDeps(41)=={18,27}
getInstDeps(140)=={119,137}
getDepDepth(84)==85
getInstDeps(90)=={-1,80}
getInstDeps(108)=={80,68}
getInstDeps(154)=={139,55}
getInstDeps(106)=={90,56}
getDepDepth(59)==95
getInstDeps(158)=={145,119}
getDepDepth(57)==85
getInstDeps(122)=={117,32}
getInstDeps(193)=={166,143}
getInstDeps(3)=={-1,-1}
getInstDeps(43)=={13,-1}
getDepDepth(190)==231
getInstDeps(141)=={140,139}
getInstDeps(87)=={55,33}
getInstDeps(10)=={-1,-1}
getDepDepth(103)==65
getInstDeps(132)=={44,116}
getDepDepth(156)==149
getDepDepth(60)==42
getInstDeps(121)=={-1,108}
getDepDepth(91)==104
getInstDeps(36)=={-1,30}
getDepDepth(7)==19
getDepDepth(63)==26
getDepDepth(69)==19
getInstDeps(36)=={-1,30}
getDepDepth(43)==23
getDepDepth(153)==177
